commit 9742abc9a9f3c3112529dbbf24dbf66ba5b02ecc
Author: Hans de Graaff <hans@winkwaves.com>
Date:   Tue Oct 20 13:20:38 2015 +0200

    Also match a capitalized password prompt
    
    sudo 1.8.12 from http://www.sudo.ws/ uses a capitalized "Password:" prompt instead of an
    all-lowercase password prompt.

diff --git a/lib/sshkit/sudo/backends_ext/netssh.rb b/lib/sshkit/sudo/backends_ext/netssh.rb
index 41cc7ec..0784699 100644
--- a/lib/sshkit/sudo/backends_ext/netssh.rb
+++ b/lib/sshkit/sudo/backends_ext/netssh.rb
@@ -30,7 +30,7 @@ module SSHKit
                   if data =~ /Sorry.*\stry\sagain/
                     SSHKit::Sudo.password_cache[password_cache_key(cmd.host)] = nil
                   end
-                  if data =~ /password.*:/
+                  if data =~ /[Pp]assword.*:/
                     key = password_cache_key(cmd.host)
                     pass = SSHKit::Sudo.password_cache[key]
                     unless pass
